lib/date.nit

```diff
@@ -0,0 +1,163 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Date manipulation
+module date
+
+# The main class Date (comming soon...)
+#
+# 	var d = new Date(2014, 01, 09)
+# 	assert d.to_s == "2014/01/09"
+#
+class Date
+	super Comparable
+
+	# Years as an integer (Int) ex: 1989
+	var year: Int
+
+	# Months as an integer (Int) ex: "3" for March
+	var month: Int
+
+	# Days as an integer (Int)
+	var days: Int
+
+	# Hours as an integer (Int)
+	var hours: Int = 0
+	
+	# Minutes as an integer (Int)
+	var mint: Int = 0
+
+	# Secound as an integer (Int)
+	var sec: Int = 0
+
+	# UTC time zone
+	var t_zone = "Z"
+
+	# for set the time (hours, min, sec) 
+	init time(y, mth, d, h, m, s: Int) do
+		year = y
+		month = mth
+		days = d
+		hours = h
+		mint = m
+		sec = s
+	end
+
+	# this will initialized the current date and time
+	init today do
```

Comment:
Actually, most of the time structure already implemented in Nit use the default constructor as `today`.

See TimeT and ISODate in https://github.com/privat/nit/blob/master/lib/standard/time.nit.